README.md

```diff
@@ -1,2 +1,89 @@
 # steampipe-mod-tailscale-compliance
 Run individual controls or full compliance benchmarks across all of your Tailscale resources using Steampipe.
+
+Run checks in a dashboard:
+
+![image](https://raw.githubusercontent.com/turbot/steampipe-mod-tailscale-compliance/main/docs/security_best_practices_dashboard.png)
+
+Or in a terminal:
+
+![image](https://raw.githubusercontent.com/turbot/steampipe-mod-tailscale-compliance/main/docs/security_best_practices_mod_terminal.png)
+
+## Getting started
+
+### Installation
+
+Download and install Steampipe (https://steampipe.io/downloads). Or use Brew:
+
+```sh
+brew tap turbot/tap
+brew install steampipe
+```
+
+Install the tailscale plugin with [Steampipe](https://steampipe.io):
+
+Clone:
+
+```sh
+git clone https://github.com/turbot/steampipe-mod-tailscale-compliance.git
+cd steampipe-mod-tailscale-compliance
+```
+
+### Usage
+
+Start your dashboard server to get started:
+
+```sh
+steampipe dashboard
+```
+
+By default, the dashboard interface will then be launched in a new browser
+window at https://localhost:9194. From here, you can run benchmarks by
+selecting one or searching for a specific one.
+
+Instead of running benchmarks in a dashboard, you can also run them within your
+terminal with the `steampipe check` command:
+
+Run all benchmarks:
+
+```sh
+steampipe check all
+```
+
+Run a single benchmark:
+
+```sh
+steampipe check benchmark.security_best_practices
+```
+
+Run a specific control:
+
+```sh
+steampipe check control.tailscale_device_authorization_enabled
```

Comment:
Is this control name correct? If this control is in the `security_best_practices` benchmark, I'd expect this to start with `security_best_practices_` at least, so something like `security_best_practices_device_authorization_enabled`.